build_container.sh

```diff
@@ -49,6 +49,8 @@ if [ "$ARCH" != "riscv64" ]; then
     popd
 fi
 
+# Allow pip to install packages system wide
+rm /usr/lib/python3.12/EXTERNALLY-MANAGED
 pip3 install --no-cache-dir pytest pexpect boto3 pytest-timeout && apt purge -y python3-pip
```

Comment:
pre-existing, but I'd suggest to avoid `&&` here (maybe in another commit of this PR), otherwise also if we have `set -e` the script will continue and we will not see the error as it happen.